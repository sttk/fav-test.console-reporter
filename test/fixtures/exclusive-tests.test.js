'use strict';

describe('Array', function() {
  describe('#indexOf()', function() {
    it.only('should return -1 unless present', function() {
      // ...
    });

    it('should return the index when present', function() {
      // ...
    });
  });
});

describe('Array', function() {
  describe('#indexOf()', function() {
    it.only('should return -1 unless present', function() {
      // this test will be run
    });

    it.only('should return the index when present', function() {
      // this test will also be run
    });

    it('should return -1 if called with a non-Array context', function() {
      // this test will not be run
    });
  });
});

describe('Array', function() {
  describe.only('#indexOf()', function() {
    it('should return -1 unless present', function() {
      // this test will be run
    });

    it('should return the index when present', function() {
      // this test will also be run
    });
  });

  describe.only('#concat()', function () {
    it('should return a new Array', function () {
      // this test will also be run
    });
  });

  describe('#slice()', function () {
    it('should return a new Array', function () {
      // this test will not be run
    });
  });
});

describe('Array', function() {
  describe.only('#indexOf()', function() {
    it.only('should return -1 unless present', function() {
      // this test will be run
    });

    it('should return the index when present', function() {
      // this test will not be run
    });
  });
});

