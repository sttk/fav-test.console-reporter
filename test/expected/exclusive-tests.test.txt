

  Array
    #indexOf()
      ✓ should return -1 unless present

  Array
    #indexOf()
      ✓ should return -1 unless present
      ✓ should return the index when present

  Array
    #indexOf()
      ✓ should return -1 unless present
      ✓ should return the index when present
    #concat()
      ✓ should return a new Array

  Array
    #indexOf()
      ✓ should return -1 unless present


  7 passing (6ms)

